eng/pipelines/extra-platforms/runtime-extra-platforms-wasm.yml

```diff
@@ -130,8 +130,6 @@ jobs:
       # Always run for runtime-wasm because tests are not run in runtime
       alwaysRun: ${{ parameters.isWasmOnlyBuild }}
 
-      # NOTE - Since threading is experimental, we don't want to block mainline work
-      shouldContinueOnError: true
```

Comment:
Shouldn't it be in a future PR, where we don't simultaneously enable any new tests, just change the warning to error? Are we 100% sure about the enabled tests to not fail?

Reply:
We need to start moving quicker now. This is the way how it will get noticed, rather than ignored.

Reply:
In the last 7 days we had  PR that are not connected with threading that had failures on '_Threading' lanes (both timeouts and BadExists). There is 100% probability that this change will introduce a mess

Reply:
that's a goal. To get people to start working the `[ActiveIssue]` chore, instead of ignoring like for the last 6 months.

Reply:
cc @lewing is it good time to start moving on this ?

Reply:
Yes, we need to start standing up the tests and marking the failing ones asap.  I'll let people know.

Reply:
I'm working to make it as green as possible before we merge this.